src/content-handlers/iiif/modules/uv-shared-module/css/thumbs-view.less

```diff
@@ -77,10 +77,18 @@
             }
 
             .thumbsView .thumbs .thumb .info .label {
-                float: left;
                 overflow-x: hidden;
                 text-overflow: ellipsis;
                 white-space: nowrap;
+                max-width: 100%;
+            }
+
+            .thumbsView .thumbs.truncate-labels .thumb .info .label {
```

Comment:
It looks like the truncate-labels class is on the wrong block of styles here -- can you please move it up to line 79? I think that will get the logic back to working as expected.